crates/cxx-qt-lib-headers/include/qt_types.h

```diff
@@ -43,6 +44,12 @@ struct rust::IsRelocatable<QString> : std::true_type
 };
 static_assert(QTypeInfo<QString>::isRelocatable);
 
+// Ensure that trivially destructible is correct
+// If this is false then we need to implement Drop rather than derive
+static_assert(std::is_trivially_destructible<QColor>::value);
```

Comment:
qIsRelocatable checks for this already: https://codebrowser.dev/qt6/qtbase/src/corelib/global/qtypeinfo.h.html#qIsRelocatable

Reply:
Yes but isn't this the default inferred one rather than the explicit one that can be overridden by the macros?

Eg `static_assert(qIsRelocatable<QString>());` fails but `static_assert(QTypeInfo<QString>::isRelocatable);` passes

Reply:
I think @ahayzen-kdab is correct here, qIsRelocatable only checks for this if it isn't overwritten elsewhere, which is still possible.